package.json

```diff
@@ -7,16 +7,28 @@
   },
   "main": "dist/index.js",
   "types": "types/index.d.ts",
+  "files": [
```

Comment:
I think that "LICENSE" and "README.md" are necessary to "files".

Reply:
According to the package.json documentation, they will be included by default.

> Certain files are always included, regardless of settings:
>
> * package.json
> * README
> * CHANGES / CHANGELOG / HISTORY
> * LICENSE / LICENCE
> * NOTICE
> * The file in the "main" field

Reply:
Oh, that's right. I get it.